### ISSUE TITLE
[11.0] website_theme_flexible vs. website_cookie_notice

### ISSUE THREAD

**Issue by ghost** [OCA/website-themes#10]
The theme denied the cookie-notice from being displayed.
Instead the content of the website is dragged "under" the menubar

**Comment by pedrobaeza** [comment]
The menubar only appears when you are logged in as website editor, so I don't think it's a big deal.

**Comment by ghost** [comment]
no. with the cookie_notice module it happens too. and the cookie_notice is hidden so you cant confirm the message
